vulnerabilities/package_managers.py

```diff
@@ -515,12 +515,12 @@ async def fetch(self, pkg, session):
 class GoproxyVersionAPI(VersionAPI):
 
     package_type = "golang"
-    pkg_mappings = {}
+    module_name_by_package_name = {}
 
     @staticmethod
     def trim_url_path(url_path: str) -> Optional[str]:
         """github advisories for golang is using package names(e.g. https://github.com/advisories/GHSA-jp4j-47f9-2vc3), yet goproxy works with module names(see https://golang.org/ref/mod#goproxy-protocol).
-        this method removes the last part of a package path, and returns the remaining as the module name.
+        this method removes the last part of a package path, and returns the remaining as the module name. For example: trim_url_path("https://github.com/xx/a/b") returns "https://github.com/xx/a"
```

Comment:
Do you mind to wrap this whole docstring to a shorter length, like 80 characters?